src/test/core.ts

```diff
@@ -968,19 +982,19 @@ describe('snabbdom', function() {
         assert.equal(elm.children.length, 1);
         elm = patch(vnode1, vnode2).elm;
         assert.equal(elm.children.length, 1);
-        rm1();
+        (rm1 as any)();
         assert.equal(elm.children.length, 1);
-        rm3();
+        (rm3 as any)();
         assert.equal(elm.children.length, 1);
-        rm2();
+        (rm2 as any)();
```

Comment:
I gave up trying to understand this test because I don't have to for now. It most likely should be written in a way that doesn't upset TypeScript (used before defined).